cli/azd/cmd/testdata/TestUsage-azd-auth-login.snap

```diff
@@ -13,7 +13,7 @@ Flags
     -h, --help                                 	: Gets help for login.
         --redirect-port int                    	: Choose the port to be used as part of the redirect URI during interactive login.
         --tenant-id string                     	: The tenant id or domain name to authenticate with.
-        --use-device-code                      	: When true, log in by using a device code instead of a browser.
+        --use-device-code string               	: When true, log in by using a device code instead of a browser.
```

Comment:
Does this go away if we create a `boolPtr` with `Type() { return "bool" }`?

Reply:
I liked how the `string` ended up..  as we can now use `t/T/True/1/true/TRUE` :)  It feels good

Reply:
@vhvb1989 We can change the UI display without the actual implementation being changed, no?

Reply:
Or do you believe this is inconsistent with our bool flags are parsed by `cobra`?

Reply:
updated to `bool`.  there's no label for bool flags...  it will be just empty like any other bool